src/mailing/sender.js

```diff
@@ -1,34 +1,49 @@
 const nodemailer = require('nodemailer');
+const Email = require('email-templates');
+const path = require('path');
 
-// transport using Nodemailer
+// Create a nodemailer transport
 const transport = nodemailer.createTransport({
   service: process.env.EMAIL_SERVICE,
   auth: {
     user: process.env.EMAIL_USERNAME, // Your Outlook email address
     pass: process.env.EMAIL_PASSWORD, // Your Outlook password
   },
+  debug: true, // Enable debugging
 });
 
-// Function to send a test email (promisified)
-async function sendTestEmail(recipient) {
-  return new Promise((resolve, reject) => {
-    transport.sendMail(
-      {
-        from: process.env.EMAIL_USERNAME,
-        to: recipient,
-        subject: 'Test Email',
-        text: 'Hello, this is a test email!',
-        html: '<p>Hello, this is a test email!</p>',
-      },
-      (err, res) => {
-        if (err) {
-          reject(err);
-        } else {
-          resolve(res);
-        }
-      }
-    );
+const baseEmail = new Email({
+  message: {
+    from: process.env.EMAIL_USERNAME,
+  },
+  transport: transport, // Use the transport you created
+  subjectPrefix: process.env.NODE_ENV === 'production' ? '' : '(TEST) ',
+  views: {
+    root: path.resolve('src/mailing/templates'), // this tells email-templates to look for templates starting from this path
+    options: {
+      extension: 'ejs', // this tells email-templates that the file will end tith *.ejs
+    },
+  },
+  send: false,
+  preview: true,
+  juice: true,
+  juiceResources: {
+    webResources: { relativeTo: path.resolve('src/mailing/templates/test') },
```

Comment:
Remember that this \`baseEmail\` variable is the config for \_all\_ of our email templates. 

This means that with this code, for _all templates_, it's going to look for the styles in the `test` folder. If you want to put all the styles and js and stuff in a single folder, that's fine, but maybe in that case put it in a folder like: `src/mailing/templates/web_resources` or something like that.

Otherwise, my initial suggestion was that this would be: `path.resolve('src/mailing/templates')` . Then if you need to refer to a style that's in the test folder, you would in that EJS file do: `<link rel='stylesheet' href='./test/style.css' data-inline>`

Reply:
@akosasante thank you!! i mush have fixed it in another branch and forgot to fix it in this one. will commit now and merge